corehq/apps/reports/views.py

```diff
@@ -2037,22 +2036,6 @@ def resave_form_view(request, domain, instance_id):
     return HttpResponseRedirect(reverse('render_form_data', args=[domain, instance_id]))
 
 
-# Weekly submissions by xmlns
-def mk_date_range(start=None, end=None, ago=timedelta(days=7), iso=False):
```

Comment:
you removed my footprint from code :-o
that was my date range

Reply:
hahaha, i had that thought too